Tests/DownloadTests.swift

```diff
@@ -97,27 +97,41 @@ class DownloadResponseTestCase: BaseTestCase {
         }
     }
 
-    func testDownloadRequestWithProgress() {
+    func testDownloadRequestWithProgress2() {
```

Comment:
But as far as test names go, they should be descriptive. If there's a name conflict—again, that's a smell of duplication and an argument for consolidation.